index.html

```diff
@@ -0,0 +1,611 @@
+          </ul>
+        </aside>
+      </section>
+      <section data-section="amazing-features" id="features">
+        <h1 class="header-underline">Some Amazing Features</h1>
+        <ul class="l-flex-container">
+          <li class="flag">
+            <span class="icon-computer"></span>
+            <h2>Responsive design</h2>
+            <p>Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</p>
+          </li>
+          <li class="flag">
+            <span class="icon-lab"></span>
+            <h2>Easy to customize</h2>
+            <p>Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</p>
+          </li>
+          <li class="flag">
+            <span class="icon-airplane"></span>
+            <h2>Perfect showcase</h2>
+            <p>Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</p>
+          </li>
+          <li class="flag">
+            <span class="icon-cog"></span>
+            <h2>App store support</h2>
+            <p>Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</p>
+          </li>
+        </ul>
+      </section>
+      <section data-section="conference-support">
+        <div class="conference-text-wrapper">
+          <p class="btn-small">New</p>
+          <h1>Live conference support available</h1>
+          <p>Eleifend morbi orci velit porttitor sed imperdiet ac ullamcorper Fusce eget tortor vel magna iaculis convallis. Aeneaneros vitae metus pellentesque tincidunt.Cum sociis natoque penatibus et magnis dis parturient montes.</p>
+          <aside>
+            <h2>Free Download On</h2>
+            <div>
+              <a target="_blank" href="https://play.google.com/store/apps?hl=en">
+                <span class="icon-android"></span><br>
+                <p>Android</p>
+              </a>
+            </div>
+            <div>
+              <a target="_blank" href="https://itunes.apple.com/us/genre/ios/id36?mt=8&cid=wwa-us-kwg-features">
+                <span class="icon-apple2"></span><br>
+                <p>iOS</p>
+              </a>
+            </div>
+          </aside>
+        </div>
+        <img src="/img/portrait-display-mockup.png" alt="" title="">
+      </section>
+      <section data-section="why-appbox" id="why">
+        <h1 class="header-underline">Why Choose AppBox</h1>
+        <img src="/img/iphone-6.png" alt="Image of iPhone" title="iPhone">
+        <ul class="why-1">
+          <li>
+            <div class="why-icon-wrapper">
+              <span class="icon-cubes"></span>
+            </div>
+            <div class="why-item-wrapper">
+              <h2>App store</h2>
+              <p>Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</p>
+            </div>
+          </li>
+          <img src="/img/spacer-dots.svg">
```

Comment:
This should have been a background on the `<li>`. In principle, I avoid `<img>`s for purely decorative elements like this. Try to find another way of including them that doesn't involve markup.

This is what I came up with:

![screen shot 2015-09-27 at 8 28 20 am](https://cloud.githubusercontent.com/assets/79984/10123570/cc755a08-64f1-11e5-930f-0e26d173c6de.png)